unskript-ctl/unskript_ctl.py

```diff
@@ -1821,6 +1822,20 @@ def save_check_names(filename: str):
             f.write(name + '\n')
 
 
+def run_script(script:list[str]):
```

Comment:
Where is the output savd to?

Reply:
This is still work in progress, its not implemented yet, i can comment it out.